dev/source/docs/code-overview-adding-a-new-parameter.rst

```diff
@@ -264,10 +264,41 @@ From time to time parameters need to be altered or renamed. ArduPilot has capabi
 
     * Set the AP_PARAM_KEY_DUMP definition to "1" [here in AP_Param.h](https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Param/AP_Param.h#L36)
 
-    * Change the delay here from 1ms to 2ms [here in AP_Param::show_all](https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Param/AP_Param.cpp#L2414)
-
-    * Remove the #if / #endif from [here in Copter's system.cpp](https://github.com/ArduPilot/ardupilot/blob/master/ArduCopter/system.cpp#L262)
-
     * Start the old code in SITL and all the parameter names and their magic numbers will be displayed
 
     * Copy-paste from Copter's Parameters.cpp file's [existing parameter conversion tables like the ones done for the attitude controller's filters](https://github.com/ArduPilot/ardupilot/blob/master/ArduCopter/Parameters.cpp#L1219).
+
+.. tip::
+
+    If you are using xterm to open the debug terminal you may find that the list of parameters is too long for the terminal's history to find the parameter(s) you are looking for.  To resolve this, create a file called *.Xresources* in your 
+    home directory.  Using your preferred text editor add the following lines and save:
+
+    ::
+
+        XTerm.vt100.saveLines: 4096
+        Term.vt100.scrollBar: true
+
+    Then, add ``xrdb ~/.Xresources`` to the end of your *.bashrc* and save.  Finally, re-load your *.bashrc* file with the command ``source ~/.bashrc``.  Upon re-running SITL, the xterm window should now have a scroll bar and a longer history allowing you to 
```

Comment:
we're better off modifying the code to save the dump to a file using AP_Filesystem, maybe "pdump.txt"

Reply:
This tip has been removed from this PR